templates/zonefile.j2

```diff
@@ -9,51 +9,51 @@ $TTL 1W
 	IN	MX 10	smtp.{{ dns.clusterid }}.{{ dns.domain | lower }}.
 ;
 ; 
-ns1	IN	A	{{ helper.ipaddr }}
-smtp	IN	A	{{ helper.ipaddr }}
+ns1			IN	A	{{ helper.ipaddr }}
+smtp			IN	A	{{ helper.ipaddr }}
 ;
-{{ helper.name | lower }}	IN	A	{{ helper.ipaddr }}
+{{ helper.name | lower }}			IN	A	{{ helper.ipaddr }}
 ;
 ; The api points to the IP of your load balancer
-api		IN	A	{{ helper.ipaddr }}
-api-int		IN	A	{{ helper.ipaddr }}
+api			IN	A	{{ helper.ipaddr }}
+api-int			IN	A	{{ helper.ipaddr }}
 ;
 ; The wildcard also points to the load balancer
-*.apps		IN	A	{{ helper.ipaddr }}
+*.apps			IN	CNAME	{{ helper.name | lower }}.{{ dns.clusterid }}.{{ dns.domain | lower }}.
```

Comment:
The syntax is. `name<TAB><TAB>IN<TAB>A<TAB>address`. So if these were converted into just whitespace, you should make sure they are now tabs.

Reply:
Oops. VS Code didn't convert it into tabs.